api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -441,11 +441,44 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 	@Test
 	public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		Visit visit = makeVisit(42);
-		visit
-		        .setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
-		
+		visit.setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
```

Comment:
And also if i remember very well, it raised an issue with the codacy check, something about the line being too long.